src/main/resources/com/google/api/codegen/go/mock.snip

```diff
@@ -0,0 +1,129 @@
+@extends "go/header.snip"
+
+@snippet generate(view)
+    {@headerComment(view.fileHeader)}
+
+    package {@view.fileHeader.localPackageName}
+
+    import (
+        {@imports(view.fileHeader.imports)}
```

Comment:
Ok that makes sense and sounds familiar, I think I encountered the same thing when prototyping Go header generation.